hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -6,7 +6,9 @@
     logrotate:
         numToKeep: 200
     builders:
-        - shell: './hack/verify-linkcheck.sh'
+        - shell: |
```

Comment:
The | means what follows is a multiline string that should have the newline at the end of each line preserved. It removes indentation for you, which is nice.